rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2683,6 +2684,15 @@ def test_upgrade_db_45_to_46(user_data_dir: 'Path', messages_aggregator):
             history_event_bindings,
         ).fetchone()[0] == 1
 
+        assert cursor.execute(
```

Comment:
should you also add a COUNT for asset movements table being zero? With a comment to change it to check the 2 tables are deleted after all is done?